install/install.ps1

```diff
@@ -8,87 +8,196 @@ $script:ErrorActionPreference='Stop'
+        
+        # We use IO.Directory::Delete instead of Remove-Item because on Windows Server 2012 with PowerShell 4.0 the latter will not work.
+        [IO.Directory]::Delete($latestDirectory)
+    }
+
+    # We use a directory junction here because not all Windows users will have permissions to create a symlink.
+    # We create this junction with cmd.exe's mklink because it has a stable interface across all active versions of Windows and Windows Server,
+    # while PowerShell's New-Item has breaking changes and doesn't have the -Target param in 4.0 (the default PowerShell on Win Server 2012).
+
+    Write-Verbose "Linking ${latestDirectory} to ${extractedDirectory}"
+    cmd.exe /Q /C "mklink /J `"$latestDirectory`" `"$extractedDirectory`"" > $null
+    if (!$?) {
+        Write-Error "Linking failed!"
+    }
+
+    Write-Verbose "Removing ${zipFile}"
+    Remove-Item -Force $zipFile
+
+    # Permanently add the latest directory to the current user's $PATH (if it's not already there).
+    # Note that this will only take effect when a new terminal is started.
+    $registryPath = 'Registry::HKEY_CURRENT_USER\Environment'
+    $currentPath = (Get-ItemProperty -Path $registryPath -Name PATH -ErrorAction SilentlyContinue).Path
+    if ($currentPath -NotMatch 'AzureAuth') {
+        Write-Verbose "Updating `$PATH to include ${latestDirectory}"
+        $newPath = if ($currentPath -Match $null) {
+            "${latestDirectory}"
+        } else {
+            "${currentPath};${latestDirectory}"
+        }
+        Set-ItemProperty -Path $registryPath -Name PATH -Value $newPath
+    }
+
+    Write-Output "Installed azureauth $version!"
 
-# We use a directory junction here because not all Windows users will have permissions to create a symlink.
-# We create this junction with cmd.exe's mklink because it has a stable interface across all active versions of Windows and Windows Server,
-# while PowerShell's New-Item has breaking changes and doesn't have the -Target param in 4.0 (the default PowerShell on Win Server 2012).
 
-Write-Verbose "Linking ${latestDirectory} to ${extractedDirectory}"
-cmd.exe /Q /C "mklink /J `"$latestDirectory`" `"$extractedDirectory`"" > $null
-if (!$?) {
-    Write-Error "Linking failed!"
 }
 
-Write-Verbose "Removing ${zipFile}"
-Remove-Item -Force $zipFile
-
-# Permanently add the latest directory to the current user's $PATH (if it's not already there).
-# Note that this will only take effect when a new terminal is started.
-$registryPath = 'Registry::HKEY_CURRENT_USER\Environment'
-$currentPath = (Get-ItemProperty -Path $registryPath -Name PATH -ErrorAction SilentlyContinue).Path
-if ($currentPath -NotMatch 'AzureAuth') {
-    Write-Verbose "Updating `$PATH to include ${latestDirectory}"
-    $newPath = if ($currentPath -Match $null) {
-        "${latestDirectory}"
+function V2-No-Latest{
+    param()
+
+    Write-Output "Using the Install No Latest Version"

```

Comment:
Ideally this is not something the user sees. We want this kind of thing to be totally transparent to the user. They don't care which installation process we use, only that when it's done the binary is on their `$PATH` and they can run `azureauth`.